core/jvm/src/test/scala/zio/logging/internal/WriterProviderSpec.scala

```diff
@@ -0,0 +1,20 @@
+package zio.logging.internal
+
+import zio.test._
+
+import java.nio.file.FileSystems
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
+object WriterProviderSpec extends ZIOSpecDefault {
+
+  val spec: Spec[Environment, Any] = suite("WriterProvider")(
+    test("Make path include date") {
+      val destination = FileSystems.getDefault.getPath("/tmp/file_app.log")
```

Comment:
please add also test for paths like:

- /tmp/file_app
- /tmp/file.app.log
- /tmp/file.app.out.log

Reply:
I added test case